theme.go

```diff
@@ -31,7 +31,7 @@ func (t *theme) themeColor(index int64, tint float64) string {
 	baseColor := t.colors[index]
 	if tint == 0 {
 		return "FF" + baseColor
-	} else {
+	} else if baseColor != "" {
```

Comment:
It's not clear that this won't create unexpected errors.   See my comment above.  The way in v3 would be to define the system colours in a lookup table and insert those values when lastClr is missing.